### PR TITLE
EVAKA-4172 Mobile staff attendance E2E tests

### PR DESCRIPTION
**Pull request by akheron**

### CODE REVIEW

File: frontend/src/e2e-playwright/pages/mobile/mobile-nav.ts [hunk 1]
```diff
@@ -0,0 +1,53 @@
+// SPDX-FileCopyrightText: 2017-2020 City of Espoo
```
Comment (mikkopiu) [inline frontend/src/e2e-playwright/pages/mobile/mobile-nav.ts:1]: Supernit: should be 2021 (the helper script should have the correct date but I assume this is from your IDE?)

File: frontend/src/e2e-playwright/specs/6_mobile/staff.spec.ts [hunk 2]
```diff
@@ -0,0 +1,138 @@
...
+    await staffPage.decStaffOtherCount()
+    await waitUntilTrue(() => staffPage.buttonsDisabled)
+  })
+
+  test('Change between groups', async () => {
+    await staffPage.incStaffCount()
+    await staffPage.confirm()
+
+    // jest.setTimeout(10000000)
+    // await page.pause()
```
Comment (mikkopiu) [inline frontend/src/e2e-playwright/specs/6_mobile/staff.spec.ts:128]: Left by accident?

**Review by mikkopiu — CHANGES_REQUESTED** [review]

File: frontend/src/e2e-playwright/pages/mobile/mobile-nav.ts (hunk 1 above)
Comment (akheron) [reply frontend/src/e2e-playwright/pages/mobile/mobile-nav.ts]: Fixed! Copy-pasted from some other file :see_no_evil:

File: frontend/src/e2e-playwright/specs/6_mobile/staff.spec.ts (hunk 2 above)
Comment (akheron) [reply frontend/src/e2e-playwright/specs/6_mobile/staff.spec.ts]: Yes, removed

**Review by mikkopiu — APPROVED** [review]
LGTM

**Pull request merged**